content-scripts/prototype-handler.js

```diff
@@ -14,6 +14,7 @@ function injectPrototype() {
       Object.prototype.hasOwnProperty.call(args[0], "runtime")
     ) {
       onceMap.vm = args[0];
+      window.__scratchAddonsTraps.dispatchEvent(new Event("gotvm"));
```

Comment:
Probably not the proper way to do it

Reply:
cc @easrng

Reply:
Why not? It's simple and it works.

Reply:
VM should be available asap for runAtComplete addons, other addons can listen for redux
I think I had that code until 1.5 or so when we refactored addon stuff, plz don't add it back

Reply:
This is used in addon-api, how would I use redux for this?

Reply:
`init` should be something addons call explicitly. Not all editor addons add blocks.

Reply:
@easrng It would be great if you could fix this.